Modern/utilities/src/main/java/org/bonej/utilities/SharedTable.java

```diff
@@ -126,13 +125,12 @@ public static void add(final String label, final String header,
 	 *
 	 * @return the singleton table.
 	 */
-	public static Table<DefaultColumn<String>, String> getTable() {
+	public static Table<DefaultColumn<Double>, Double> getTable() {
 		return table;
 	}
 
 	public static boolean hasData() {
-		return table.stream().flatMap(Collection::stream).anyMatch(s -> s != null &&
-			!EMPTY_CELL.equals(s));
+		return table.stream().flatMap(Collection::stream).anyMatch(s -> s != null );
```

Comment:
Could use method reference `Objects::nonNull` instead of lambda expression

Reply:
`anyMatch(Objects::nonNull)`

Reply:
Example, please?

Reply:
Never mind, got it.